dnscrypt-proxy/example-dnscrypt-proxy.toml

```diff
@@ -444,10 +444,10 @@ cache_neg_max_ttl = 600
 
 
 ######################################################
-#        Pattern-based blocking (blacklists)        #
+#        Pattern-based blocking (blocked lists)        #
```

Comment:
I think it would be clearer if in the comments we consistently refer to blocklists and allowlists to describe the abstract concept, then use the "blocked" and "allowed" terminology for specific config options (`blocked_names`, `blocked_ips`, `allowed_names`). Right now the comments are a bit mixed between "blocklist" and "blocked lists", which is a bit confusing. I am not too fussed which one we choose, just as long as we are consistent.

From my side, I think "blocklist" is clearer as a noun than "blocked list", since to me "blocked list" sounds like the list itself is blocked, rather than that it is a list containing blocks.

Reply:
Agreed and updated in latest PR update